test_scripts/run

```diff
@@ -23,6 +23,7 @@ TESTS="./build/test/bin/cbortest \
        ./test_scripts/tree0 \
        ./test_scripts/tree1 \
        ./test_scripts/tree2 \
+       ./test_scripts/tutorial \
```

Comment:
Yes, tutorial code is built with the rest of the code keeping in sync. Love this.